Makefile

```diff
@@ -50,9 +62,9 @@ $(OUT)/cpu_instrs.h: tests/cpu_instrs.gb tests/rom2h.c
 	$(Q)$(CC) -o $(OUT)/rom2h tests/rom2h.c
 	@$(OUT)/rom2h
 
-$(OUT)/bench: $(OUT)/cpu_instrs.h prof.h cpu.c bench.c gameboy.h
+$(OUT)/bench: $(GBIT_OBJS) $(OUT)/cpu_instrs.h prof.h cpu.c bench.c gameboy.h
 	$(VECHO) "  CC+LD\t$@\n"
-	$(Q)$(CC) -DENABLE_LCD=0 -o $@ cpu.c bench.c
+	$(Q)$(CC) -DENABLE_LCD=0 -DDEBUG -o $@ cpu.c bench.c $(GBIT_OBJS)
```

Comment:
Remove macro `DEBUG` since the benchmark program might be built for release rather than debugging.